pkg/kv/kvnemesis/kvnemesis_test.go

```diff
@@ -54,6 +56,74 @@ func testClusterArgs(tr *SeqTracker) base.TestClusterArgs {
 		},
 	}
 
+	isOurCommand := func(ba *kvpb.BatchRequest) (string, uint64, bool) {
+		key := ba.Requests[0].GetInner().Header().Key
+		n, err := fkE(string(key))
```

Comment:
Should this also check that the key belongs to the data span?

Reply:
I thought `fkE` fully did it, it actually didn't, but adding a commit that makes it so.